agent/agent.go

```diff
@@ -445,13 +435,19 @@ func (a *Agent) finishTask(ctx context.Context, tc *taskContext, status string)
 		tc.logger.Task().Error("Task completed - CANCELED.")
 		// If we receive a 409, return control to the loop (ask for a new task)
 		return nil, nil
+	case evergreen.TaskSystemFailed:
+		grip.Error("Task system failure")
```

Comment:
this just goes to the agent logfile? is this what you want?

Reply:
Yes, because if this happens that means there is no logger available